csharp-client-reference/IEvaluator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using Google.Protobuf.Collections;
+
+namespace EmpowerOps.Volition.RefClient
+{
+    public interface IEvaluator
```

Comment:
This is the only one newlly added, not to much value now but i think it is a good direction to chase